framework/src/controller/channels/child_process_channel.js

```diff
@@ -0,0 +1,137 @@
+const { EventEmitter2 } = require('eventemitter2');
+const axon = require('axon');
+const { Server: RPCServer, Client: RPCClient } = require('axon-rpc');
+const Action = require('../action');
+const Event = require('../event');
+const BaseChannel = require('./base_channel');
+
+/**
+ * Channel responsible to communicate with bus for modules running in child process
+ *
+ * @class
+ * @memberof framework.controller.channels
+ * @requires module.Event
+ * @requires module.Action
+ * @requires channels/base_channel
+ * @type {module.ChildProcessChannel}
+ */
+class ChildProcessChannel extends BaseChannel {
```

Comment:
What do you think of `CrossProcessChannel`? For me, this way seems more informative, describing that the channel works across different processes.

Reply:
I agree, but let's leave as it is for now and change it when we have a better distinction of both `ChildProcessChannel` and `InMemoryChannel`.